src/coreclr/src/jit/instrsxarch.h

```diff
@@ -594,6 +594,7 @@ INST3(blsr,             "blsr",             IUM_WR, BAD_CODE,     BAD_CODE,
 INST3(bextr,            "bextr",            IUM_WR, BAD_CODE,     BAD_CODE,     SSE38(0xF7),                             INS_Flags_IsDstDstSrcAVXInstruction)    // Bit Field Extract
 
 // BMI2
+INST3(rorx,             "rorx",             IUM_WR, BAD_CODE,     BAD_CODE,     SSE3A(0xF0),                             INS_FLAGS_None)
```

Comment:
It would be good to double-check that neither the `IsDstDstSrcAVXInstruction` nor `IsDstSrcSrcAVXInstruction` flags apply. The instruction looks to be a `dst, src, imm` operand in this case and so it shouldn't be flagged as RMW and should go down the right encoding paths in `emit`.

Reply:
tried both of these flags and observed SEH exceptions on start 😞